magento/magento2
# Document Exceptions in Asset Repository Functions

### Description (*)
The `updateDesignParams()` function has the capacity to throw the `UnexpectedValueException` but this is not documented in either of the functions that call it. Additionally, the `createAsset()` function has the potential to throw a `LocalizedException` which is not documented by either of the functions that make use of it.

### Manual testing scenarios (*)
1. Create a class that makes use of the modified functions
2. Observe your IDE report the potential for the aforementioned exceptions

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38988: Document Exceptions in Asset Repository Functions
